reports/accidents_count_phenomenon_per_hour.csv

```diff
@@ -0,0 +1,6 @@
+Phenomenon,Accident_count,total_phenomenon_hours,Accident_per_hour

```

Comment:
This files seems to be generated, do you really need to have them pushed in the repo?

Reply:
Removed, thanks